liquid

```diff
@@ -4,10 +4,16 @@ set -e
 # cd to file
 cd "$(dirname ${BASH_SOURCE[0]})"
 
+# delete venv if outdated
+if ! shasum -c venv/requirements.shasum; then
```

Comment:
`if ! shasum -c venv/requirements.shasum requirements.txt; then` right?

Reply:
No, `requirements.shasum` contains checksums and filenames.  Well, a single checksum, and a single filename. But that filename is exactly `requirements.txt`. :)